twisted/internet/endpoints.py

```diff
@@ -506,7 +506,7 @@ def __init__(self, reactor, port, backlog=50, interface='::'):
         @param backlog: Size of the listen queue
         @type backlog: int
 
-        @param interface: The hostname to bind to, defaults to '' (all)
+        @param interface: The hostname to bind to, defaults to '::' (all)
```

Comment:
the actual markup shoud be `default to C{::} (all)` as C{} is the markup used for exact code